charts/tekton-pipelines/templates/common/tasks/slack-notification.yaml

```diff
@@ -102,13 +102,26 @@ spec:
         argo_app_url=$(echo $argo_app_url | sed 's/\/\//\//g')
         argo_app_url="https://$argo_app_url"
 
+        # Preparing jira links
+        # We are using grep as ash doesn't support regex test
+        jira_task=$(echo "$(params.head_commit_message)" | grep -oE '[A-Z0-9]+-[0-9]+' | head -n 1 )
+        if [ "$jira_task" != "" ];
+        then
+          echo "Found $jira_task jira task, preparing link"
+          head_commit_message=$(echo "$(params.head_commit_message)" | sed "s|$jira_task|\<https://saritasa.atlassian.net/browse/$jira_task/\|$jira_task\>|")
```

Comment:
Please add a screenshot, how it looks like as a result?

Reply:
Done